R/rj.b.R

```diff
@@ -106,6 +109,13 @@ RjClass <- R6::R6Class(
             }
 
         },
+        .setZipEnv=function() {
+            zip <- Sys.getenv('R_ZIPCMD')
+            if ( substr(zip, (nchar(zip)+1)-7, nchar(zip)) != "zip.exe" ) {
+                zip <- system.file("exec/zip.exe", package = "Rj")
```

Comment:
don't we want to go `system.file( ..., package='zip')`? ... rj doesn't contain `zip.exe`, so this won't work.

Reply:
Hi Jth,
this PR for Rj, are my final conclusions of what we have already discussed in SLACK.
To allow the Rj environment (both bundle or external) to solve the problem (which occurs only in Windows) when there are calls like zip {utils} is to make it find the zip command in a path that can also be at the level user, R etc. is not necessarily installed.
Among the various solutions tried, that of carrying the zip.exe file with Rj (exec folder) has proved to be the lightest that has never failed or left "environmental" garbage.
However, if you think this proposed final solution is not aligned, close it and leave things as they are.

Reply:
sorry, i thought the solution was to add the `zip` package as a dependency to rj ... at the moment, there's no mechanism for placing `zip.exe` inside `rj/exec` where as the `zip` package is already shipping with `zip.exe` so it makes sense to use that.

sorry for the misunderstanding

cheers